lib/core/src/lib/buttons-menu/buttons-menu.component.stories.ts

```diff
@@ -1,119 +0,0 @@
-/*!
```

Comment:
Same question for stories

Reply:
we don't have storybook for Insights as it's the unsupported and outdated library